src/app/Main/Body/QueryPage/Results/TableResults/TableResults-TableMainHeader.component.tsx

```diff
@@ -38,15 +38,15 @@ const tableTitleStyle = (
     transform: ${isRow ? "rotate(180deg)" : "none"};
 `;
 
-interface TestInterface {
+interface TableMainHeaderInterface {

```

Comment:
TableMainHeaderProps?

Reply:
Yes, it was changed in the commit "Unify names of interfaces"